src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -369,6 +419,26 @@ export class InputTimePicker
   //
   // --------------------------------------------------------------------------
 
+  private delocalizeTimeString(value: string): string {
+    // Parsing won't work with the correct locale unless you call this right before dayjs()
```

Comment:
Suggestion: `we need to set the corresponding locale before parsing, otherwise it defaults to English (possible dayjs bug)`